src/js/components/Nav/stories/CustomChild.js

```diff
@@ -30,10 +30,10 @@ const SidebarButtons = () => {
               onClick={() => setActive(label)}
             />
           ))}
-        </Box>
+        </Nav>
       </Box>
     </Grommet>
   );
 };
 
-storiesOf('Button', module).add('Sidebar', () => <SidebarButtons />);
+storiesOf('Nav', module).add('Custom child', () => <SidebarNav />);
```

Comment:
Is it a "custom" child? Or is it just a normal use of Nav?

Reply:
I've replaced it with the name `Sidebar`, it doesn't make sense to have `Custom child` when we don't have `items`.